Docs/install/index.rst

```diff
@@ -34,118 +34,175 @@
    #  @@-COPYRIGHT-END-@@
    # =============================================================================
 
+.. role:: hideitem
+   :class: hideitem
+
 .. _ug-installation:
 
-###################
+##################
 AIMET Installation
-###################
+##################
 
-Quick Install
-~~~~~~~~~~~~~
+This page contains instructions for the following installation scenarios:
+
+- :ref:`Quick Install of the latest version for PyTorch <idx-install-quick>`
+- :ref:`Installation of the latest version for all platforms from release packages <idx-install-latest>`

```

Comment:
I feel the title is tool long. At a minimum, we should remove `from release packages` since that term is unclear.

Reply:
Removed.